src/EditorFeatures/CSharpTest/CodeActions/IntroduceVariable/IntroduceVariableTests.cs

```diff
@@ -3172,7 +3172,31 @@ class B
 
         [WorkItem(528, "http://github.com/dotnet/roslyn/issues/528")]
         [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsIntroduceVariable)]
-        public async Task TestIntroduceLocalInExpressionBodiedMethod()
+        public async Task TestIntroduceFieldInExpressionBodiedMethod()
```

Comment:
I actually reordered this test (TestIntroduce**Field**InExpressionBodiedMethod, TestIntroduce**Local**InExpressionBodiedMethod) and another test down below, because the sequence in the tests was: local field, local field and then the rest of the tests is field local, field local. I hope it's not too bad in the diff.